MangaManager/src/MetadataManager/MetadataManagerLib.py

```diff
@@ -209,15 +211,13 @@ def preview_export(self, loaded_cinfo):
         # loaded_cinfo.cinfo_object.export(export, 0)
         # print(export.getvalue())
```

Comment:
```suggestion
```

Reply:
This isn't related to my changes, I didn't remove any dead code you already had present. Do you want me to proceed or leave this for another PR?